backup_and_restore/backing-up-etcd.adoc

```diff
@@ -0,0 +1,30 @@
+[id="backup-etcd"]
+= Backing up etcd
+include::modules/common-attributes.adoc[]
+:context: backup-etcd
+
+toc::[]
+
+etcd is the key value store for all object definitions, as well as the
+persistent master state. Other components watch for changes, then bring
+themselves into the desired state.
```

Comment:
small nit could we clarify the first sentence in this paragraph and maybe drop the second?

`etcd is the key value store for OpenShift which persists state of all resource objects.`